app/models/reservation_hold.rb

```diff
@@ -20,11 +21,7 @@ class ReservationHold < ApplicationRecord
 
   # Total number of items that were reserved
   def loaned_quantity
-    if item_pool.uniquely_numbered?
-      reservation_loans.size
-    else
-      reservation_loans.pluck(:quantity).sum
-    end
+    reservation_loans.sum(:quantity)
```

Comment:
This is an example of the simplification we get from always setting `quantity`, even for individual items.